src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -1040,6 +1040,63 @@ MulticopterAttitudeControl::task_main()
 					_controller_status_pub = orb_advertise(ORB_ID(mc_att_ctrl_status), &_controller_status);
 				}
 			}
+
+			if (_v_control_mode.flag_control_termination_enabled) {
```

Comment:
This where I have stared based on the FW_att_control but I couldn't stop motors from here. 
For lockdown we don't really need anything, but for termination it is the a kind of only place to do something.
 IMO drivers should not know about flight control modes and take actions based on this.